variables.py

```diff
@@ -39,9 +39,29 @@
     #'byMediumIsolationMVA3oldDMwLT':{'var':'tau_byMediumIsolationMVA3oldDMwLT > 0.5 && tau_decayModeFindingOldDMs > 0.5', 'nbin':2, 'min':-0.5, 'max':1.5, 'title':'byMediumIsolationMVA3oldDMwLT'},
     #'byTightIsolationMVA3oldDMwLT':{'var':'tau_byTightIsolationMVA3oldDMwLT > 0.5 && tau_decayModeFindingOldDMs > 0.5', 'nbin':2, 'min':-0.5, 'max':1.5, 'title':'byTightIsolationMVA3oldDMwLT'},
 
-    'byLooseIsolationMVArun2v1DBoldDMwLT': {'var': 'tau_byLooseIsolationMVArun2v1DBoldDMwLT > 0.5 && tau_decayModeFindingOldDMs > 0.5', 'nbin': 2, 'min': -0.5, 'max': 1.5, 'title': 'byLooseIsolationMVArun2v1DBoldDMwLT'},
+    'byLooseIsolationMVArun2v1DBoldDMwLT':  {'var': 'tau_byLooseIsolationMVArun2v1DBoldDMwLT  > 0.5 && tau_decayModeFindingOldDMs > 0.5', 'nbin': 2, 'min': -0.5, 'max': 1.5, 'title': 'byLooseIsolationMVArun2v1DBoldDMwLT'},
     'byMediumIsolationMVArun2v1DBoldDMwLT': {'var': 'tau_byMediumIsolationMVArun2v1DBoldDMwLT > 0.5 && tau_decayModeFindingOldDMs > 0.5', 'nbin': 2, 'min': -0.5, 'max': 1.5, 'title': 'byMediumIsolationMVArun2v1DBoldDMwLT'},
-    'byTightIsolationMVArun2v1DBoldDMwLT': {'var': 'tau_byTightIsolationMVArun2v1DBoldDMwLT > 0.5 && tau_decayModeFindingOldDMs > 0.5', 'nbin': 2, 'min': -0.5, 'max': 1.5, 'title': 'byTightIsolationMVArun2v1DBoldDMwLT'},
+    'byTightIsolationMVArun2v1DBoldDMwLT':  {'var': 'tau_byTightIsolationMVArun2v1DBoldDMwLT  > 0.5 && tau_decayModeFindingOldDMs > 0.5', 'nbin': 2, 'min': -0.5, 'max': 1.5, 'title': 'byTightIsolationMVArun2v1DBoldDMwLT'},
```

Comment:
Seeing so many insertions with the same ranges, selections, etc, it would be good to use a loop over WPs 
['Loose', 'Medium', 'Tight', etc], and over MVA names, instead of writing everything explicitly.

Reply:
I could do it but rewriting only the part with different mvaIDs will not be much shorter than specifying them explicitly. In principle, this file has to be rewritten and these two things can be done in a single iteration in the next PR just not to do the same work twice